### PR TITLE
Japanese translation

### PR DESCRIPTION
**Pull request by ndoll**
There might be a better term for Guaranteed Min, but the rest should be solid.

### CODE REVIEW

**Comment by DevSplash** [comment]
Please update `/src/i18n.js` and  `/src/Localizer.jsx`. 

> In /src/i18n.js, create an import statement for your language, importing the translation.json file you just created. Follow the pattern used by the existing statements.

>In the same file (/src/i18n.js), add your language to the list of languages inside the resources object within the i18n.init() method, following the pattern of other declared languages, and using the same two-letter language tag you used for the folder name in the first step.

>Lastly, to add your language to the manual language switcher on the page, make sure to edit the Localizer component at /src/Localizer.jsx. Add a tuple for your language to the languages array, with the first value being your two-letter language code and the second value being the name of your language, in your language.

**Comment by ndoll** [comment]
Sorry about that, all done!

**Comment by mtaylor76** [comment]
Small nitpick, but make sure to use the correct [ISO 639-1](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes) code for Japanese, which is `jo`. 

There's a slight error in that you use the code `jp`, which is the code for **Japan,** but not for **Japanese**. We're trying to keep the standard to language codes, not country codes.

File: locales/jp/translation.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
```
Comment (mtaylor76) [inline locales/jp/translation.json:1]: Change locale folder name from `jp` to `jo` ([ISO 639-1 language codes](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes))

File: src/Localizer.jsx [hunk 2]
```diff
@@ -13,6 +13,7 @@ const languages = [
   ["zh-TW", "繁体中文（台湾）"],
   ["zh-HK", "繁体中文（香港）"],
   ["ko", "한국어"],
+  ["jp", "日本語"],
```
Comment (mtaylor76) [inline src/Localizer.jsx:16]: `jp` -> `jo` ([ISO 639-1 language codes](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes))

File: src/i18n.js [hunk 3]
```diff
@@ -70,6 +71,9 @@ i18n
       "zh-HK": {
         translations: translationZhHK,
       },
+      jp: {
+        translations: translationJp,
+      },
```
Comment (mtaylor76) [inline src/i18n.js:76]: `jp` -> `jo` ([ISO 639-1 language codes](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes))

**Comment by DevSplash** [comment]
> Small nitpick, but make sure to use the correct [ISO 639-1](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes) code for Japanese, which is `jo`.
> 
> There's a slight error in that you use the code `jp`, which is the code for **Japan,** but not for **Japanese**. We're trying to keep the standard to language codes, not country codes.

The Japanese language code seems to be `ja` instead of `jo`.

**Comment by mtaylor76** [comment]
> 
> 
> > Small nitpick, but make sure to use the correct [ISO 639-1](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes) code for Japanese, which is `jo`.
> > There's a slight error in that you use the code `jp`, which is the code for **Japan,** but not for **Japanese**. We're trying to keep the standard to language codes, not country codes.
> 
> The Japanese language code seems to be `ja` instead of `jo`.

Err, yeah, you're right about that. Been working on fixing the Korean translation at the same time (which is `ko` instead of `kr`) and mixed up the letters. I'll fix that, good catch.

**Comment by ndoll** [comment]
Thanks for pointing that out, I'll make the changes now.

**Comment by elxris** [comment]
Thank you! #17

**Pull request merged**